### ISSUE TITLE
[Swift 5.0]: VerticalParameterAlignmentOnCallRule is broken

### ISSUE THREAD

**Issue by jpsim** [realm/SwiftLint#2606]
```swift
myFunc(foo: 0,
       bar: baz == 0)
            ^ warning: Vertical Parameter Alignment On Call Violation: Function parameters should be aligned vertically if they're in multiple lines in a method call. (vertical_parameter_alignment_on_call)
```

This doesn't violate for Swift 4.x but reports an incorrect violation in Swift 5 bundled with Xcode 10.2 beta 1.

I filed [SR-9776](https://bugs.swift.org/browse/SR-9776) but we may want to consider alternative "creative" solutions if that doesn't get fixed for Swift 5.0.

**Comment by marcelofabri** [comment]
The good news is that it seems to be broken only when using operators?

**Comment by jpsim** [comment]
Yeah, other operators and ternary expressions too.

**Comment by jpsim** [comment]
Closing, this has been fixed in Swift's master and Swift 5 branches.

**Comment by tonyarnold** [comment]
I'm seeing this with Swift 4.2 running SwiftLint 0.30.1 - is my Mac being special again?

**Comment by jpsim** [comment]
> I'm seeing this with Swift 4.2 running SwiftLint 0.30.1 - is my Mac being special again?

Maybe? Hard to say without steps to reproduce 😅

**Comment by tonyarnold** [comment]
I can't reproduce it consistently. I did have Xcode 10.2b1 set as my command-line tools, so maybe that's causing chaos. Sorry, I'll come back when I have something concrete to give you.

**Comment by jpsim** [comment]
SwiftLint does look at `DEVELOPER_DIR` to know what version of Swift to use to lint. So if Xcode 10.2b1 was set to your xcode-select, you'd get this from the CLI but not when linting from Xcode 10.1 GUI as a build phase.

**Comment by tonyarnold** [comment]
No, this is definitely happening from within a build phase. The build phase calls out to a bash script that calls SwiftLint, but I would assume it's inheriting the environment passed to it by Xcode.

**Comment by jpsim** [comment]
> I would assume it's inheriting the environment passed to it by Xcode.

It should! Though again, it's all speculation until we get more info.

**Comment by tonyarnold** [comment]
Running with `--no-cache` in the script fixed the issue. Something must have been retained when I switched back and forth between Xcode 10.1 & 10.2b1. Thanks as always, JP - you're a gentleman.
